### ISSUE TITLE
Missing notification dot icon in mobile mode

### ISSUE THREAD

**Issue by SeanPM5** [maykar/compact-custom-header#175]
## Describe the issue:

With the recent redesigns in HA 0.96 the only way to see if you have notifications without opening the sidebar (on a **mobile device**) is the orange dot that displays over the hamburger icon. 

But this does not seem to work on CCH (unless I missed something). I see it for a split second while refreshing and HA's menu loads but once it gets replaced with the CCH version of the menubar, that orange icon goes away. Would be nice if that could be added.

Here's a comparison of what I mean (HA left, CCH right):

![IMG_1371](https://user-images.githubusercontent.com/9065557/61603443-8eaf9480-ac0b-11e9-83ca-eb7fd6c6814a.PNG)

## Config YAML:

```yaml
cch:
  hide_tabs: '3,4'
  voice: hide
  menu: show
  chevrons: false

  swipe: true
  swipe_animate: none
```
## HA and/or Browser Error:

No errors

## Versions and Browser:

HA 0.96.3
CCH 1.3.1
Official HA iOS 2.0 app
YAML mode

N/A

**Comment by bagobones** [comment]
Just commenting to follow.

**Comment by maykar** [comment]
Fixed in 1.3.2

**Comment by SeanPM5** [comment]
Thanks for fixing it. However I think there’s one tiny issue. Just updated to CCH 1.3.2 and it shows like this on the iOS 2.0b app (not home at the moment to test on desktop):

![alt text](https://i.imgur.com/OUb6JHs.jpg)

You can see the outline/border of the circle in the top right of the hamburger menu, but the dot seems empty and not filled in with the accent-color.

**Comment by maykar** [comment]
Looks like this one is going to be a bit more of a pain than I realized. Not only does it seem that the HA renders the notification dot differently on mobile than it does on PC, but differently on iOS vs android.

**Comment by maykar** [comment]
Just pushed a quick update 1.3.3 to address. Let me know if it helps.

**Comment by SeanPM5** [comment]
Yep it's working perfectly now on iOS (both the stable 1.5.1 and 2.0 beta). Many thanks!

**Comment by bagobones** [comment]
Working on iOS for me now. 1.3.3

On desktop I have "Always hide the sidebar" set in hass, would it be possible to display the dot for desktop in this case as well?

**Comment by maykar** [comment]
That looks to be a bug/feature request for Home Assistant itself and should be reported/handled there, but you could make an indicator using CCH. Something like this:

```yaml
cch:
  conditional_styles:
    - entity: notifications
      condition:
        state: true      # When there are notifications...
      button:             # color menu button yellow.
        menu:
          color: yellow
```

You could even change the icon to a bell or something similar, more here:

https://maykar.github.io/compact-custom-header/Conditional-Styling-Config/

**Comment by maykar** [comment]
Just to be more clear, when using "Always hide the sidebar" HA doesn't provide the notification dot. This is not caused by CCH and should be reported on the HA github. It's not a good idea to fix bugs in home assistant with custom projects.

**Comment by SeanPM5** [comment]
Which version of Home Assistant are you on @bagobones ? I believe this was added in a later point release of 0.96. 

Cause if you visit https://demo.home-assistant.io/ on a desktop browser and click the profile page there and turn on the "Always hide the sidebar" option it does show the notification dot over the menu on desktop. (The demo is running 0.97 beta but I am fairly sure it was added in 0.96.4 or 0.96.5).

So you might have to either update HA or hard refresh the browser.

**Comment by maykar** [comment]
I'm on 0.96.5 and I still see that behavior, could be that it was fixed in the beta

**Comment by maykar** [comment]
Just updated to 0.97.0b1 and it is fixed there. It does however add a border of the primary color that I'll need to adjust.

**Comment by bagobones** [comment]
I am running HA 0.96.5, I have noticed the icon appears if I resize my browser window small enough.

So it is an HA bug fixed in 0.97.0b1  @maykar ? I guess I will let it bake some more.

Also is it a bug or just a not fully realized UI change.. I think "FIXING" the default UI is exactly what a lot of custom components do..

**Comment by maykar** [comment]
Yeah, it's fixed in 0.97.0b1. I think this was something that was just overlooked when "Always hide the sidebar" was added.

The new sidebar is still pretty fresh and going through a bunch of changes. No point in adding things like this to CCH when they can change in the next HA version.

CCH messes with a ton of different elements of the interface and there's a chance any of them can have changes with each new version. This leads to me doing a good bit of catch up and I'd like to keep that to a minimum.

I need to stay away from things that should be fixed in core and this is just one of those things. I wouldn't say CCH really "fixes" anything, but it does add features that are either a ways off from being added (like per user views) or will never be added (like conditional styling of the header).
